source/api_documentation/transactions.rst

```diff
@@ -1,792 +0,0 @@
-
-  * - dynamic_number_pool_referrer_param5_value
-    - Pool Param 5 Value
-    - [Number pooling links only] The value for parameter 5.
-
-  * - dynamic_number_pool_referrer_param6_name
-    - Pool Param 6 Name
-    - [Number pooling links only] The name for parameter 6.
-
-  * - dynamic_number_pool_referrer_param6_value
-    - Pool Param 6 Value
-    - [Number pooling links only] The value for parameter 6.
-
-  * - dynamic_number_pool_referrer_param7_name
-    - Pool Param 7 Name
-    - [Number pooling links only] The name for parameter 7.
-
-  * - dynamic_number_pool_referrer_param7_value
-    - Pool Param 7 Value
-    - [Number pooling links only] The value for parameter 7.
-
-  * - dynamic_number_pool_referrer_param8_name
-    - Pool Param 8 Name
-    - [Number pooling links only] The name for parameter 8.
-
-  * - dynamic_number_pool_referrer_param8_value
-    - Pool Param 8 Value
-    - [Number pooling links only] The value for parameter 8.
-
-  * - dynamic_number_pool_referrer_param9_name
-    - Pool Param 9 Name
-    - [Number pooling links only] The name for parameter 9.
-
-  * - dynamic_number_pool_referrer_param9_value
-    - Pool Param 9 Value
-    - [Number pooling links only] The value for parameter 9.
-
-  * - dynamic_number_pool_referrer_param10_name
-    - Pool Param 10 Name
-    - [Number pooling links only] The name for parameter 10.
-
-  * - dynamic_number_pool_referrer_param10_value
-    - Pool Param 10 Value
-    - [Number pooling links only] The value for parameter 10.
-
-  * - dynamic_number_pool_referrer_search_type
-    - Search Type
-    - [Number pooling links only] "Paid" or "Organic".
-
-  * - dynamic_number_pool_pool_type
-    - Pool Type
-    - [Number pooling links only] The type of pool: "Search", "SearchKeyword" or "Custom".
-
-  * - dynamic_number_pool_id
-    - Pool ID
-    - [Number pooling links only] The ID of the pool.
-
-  * - start_time_local
-    - Call Start Time
-    - Start of the call in the API user's time zone, followed by offset from GMT.
-
-  * - start_time_xml
-    - Call Start Time (XML formatted)
-    - Start of the call in Soap XML formatted time.
-
-  * - start_time_utc
-    - Call Start Time (UTC timestamp)
-    - Start of the call in milliseconds since Jan 1, 1970. Divide by 1000 to get Unix epoch time.
-
-  * - corrected_at
-    - Corrected At
-    - [Correction only] Date and time the transaction was corrected, in user's time zone, followed by offset from GMT.
-
-  * - opt_in_SMS
-    - Opt In Sms
-    - Whether the caller opted in to receive an SMS promotion.
-
-  * - complete_call_id
-    - Call Record ID
-    - Globally unique identifier for the call this transaction is part of. Up-to 32 character string, can contain alphanumeric characters (i.e. 0-9A-Z) and the "-".
-
-  * - transfer_from_type
-    - Transfer Type
-    - Where the call came from.
-
-  * - notes
-    - Notes
-    - Free-form notations on transaction.
-
-  * - call_outcome_outcome_name
```

Comment:
(just so you have a placeholder item in the PR) as discussed these call_outcome_* fields are now missing